### PR TITLE
updated manpage and added --help option to xlib main.c

### PR DESCRIPTION
**Pull request by cebe**
Could have added it for windows and mac but I have no environment to compile and test those files.

### CODE REVIEW

File: xlib/main.c [hunk 1]
```diff
@@ -1120,7 +1120,7 @@ int main(int argc, char *argv[]) {
                 return 1;
             }
             if(!strcmp(argv[i], "--version")) {
-                debug("%s\n", VERSION);
+                printf("%s\n", VERSION);
```
Comment (GrayHatter) [inline xlib/main.c]: why?

Comment (cebe) [inline xlib/main.c]: I thought that debug may be removed or silenced in stable versions or something. This output should be enabled regardless of debug mode or not.

**Comment by GrayHatter** [comment]
@cebe please change it back to debug? I have plans to add additional debug macros for different levels of verbosity

File: xlib/main.c (hunk 1 above)
Comment (GrayHatter) [inline xlib/main.c]: @cebe please change it back to debug? I have plans to add additional debug macros for different levels of verbosity

Comment (cebe) [inline xlib/main.c]: will do.

**Comment by GrayHatter** [comment]
:+1: 

@cebe this is probably okay, but using printf instead of debug breaks support for android, and likely for windows as well, as windows will buffer output, and hold it until whenever it wants.

**Comment by cebe** [comment]
as far as I see, windows has its own [main.c](https://github.com/GrayHatter/uTox/blob/develop/windows/main.c) and android too: https://github.com/GrayHatter/uTox/blob/develop/android/main.c

As far as I understood my changes do not have any effect on those platforms but I may be wrong.

**Comment by GrayHatter** [comment]
@cebe no, it won't stop them from compiling, but you can't use printf on android. I.e. your code can't be copied over to android. Not a problem as I don't see anyone needing to run --help in android.

**Comment by cebe** [comment]
I think it if fine this way then.

**Comment by stal888** [comment]
@GrayHatter your call.

**Pull request merged**